src/commands/android/utils/adbWirelessConnect.ts

```diff
@@ -0,0 +1,73 @@
+import {Platform} from '../interfaces';
+import {execBinarySync} from './sdk';
+import Logger from '../../../logger';
+import inquirer from 'inquirer';
+import colors from 'ansi-colors';
+
+export async function connectAdbWirelessly(adbLocation: string, platform: Platform): Promise<boolean> {
+  try {
+    const deviceList = execBinarySync(adbLocation, 'adb', platform, 'devices -l');
+    if (deviceList !== null) {
+      if (!deviceList.toString().includes('transport_id')) {
+        Logger.log(colors.red('No devices found. Please connect a device and try again.'));
+
+        return false;
+      }
+    }
+
+    Logger.log(`${colors.bold('Follow the below steps to connect to your device wirelessly:\n')}
+  1. Enable USB debugging on your device.
+  2. Connect your device to your computer using a USB cable.
+  3. Grant permission for File transfer on your device.
+  4. Connect your device to the same network as your computer ${colors.grey('(you may connect your device to your computer\'s hotspot)')}.
+  5. Enable wireless debugging on your device.
+  6. Find the IP address, port number and pairing code of your device by going to 
+     ${colors.cyan('Settings > Wireless Debugging > Pair device with pairing code.')}\n`);
+
+    const deviceIPAnswer = await inquirer.prompt({
+      type: 'input',
+      name: 'deviceIP',
+      message: 'Enter the IP address of your device:'
+    });
+    const deviceIP = deviceIPAnswer.deviceIP;
+
+    const portAnswer = await inquirer.prompt({
+      type: 'input',
+      name: 'port',
+      message: 'Enter the port number:'
+    });
+    const port = portAnswer.port;
+
+    const pairingCodeAnswer = await inquirer.prompt({
+      type: 'input',
+      name: 'pairingCode',
+      message: 'Enter the pairing code displayed on your device:'
+    });
+    const pairingCode = pairingCodeAnswer.pairingCode;
+    const pairing = execBinarySync(adbLocation, 'adb', platform, `pair ${deviceIP}:${port} ${pairingCode}`);
+    if (pairing) {
+      Logger.log(`${colors.green('\nPairing successful!')} ${colors.bold('Please remove the USB cable before proceeding further:\n')}`);
+    } else {
+      Logger.log(`${colors.red('\nPairing failed!')} please try again.`);
+
+      return false;
+    }
+
+    const finalPortAnswer = await inquirer.prompt({
+      type: 'input',
+      name: 'finalPort',
+      message: 'Enter new port number after disconnecting usb:'
+    });
+    const finalPort = finalPortAnswer.finalPort;
```

Comment:
We should probably ask for this port beforehand, as it appears before clicking on "Pair device with pairing code.".

We could also keep giving instructions before asking a particular question, because distinguishing between the two port numbers can be a bit technical.